nodecg-io-philipshue/extension/index.ts

```diff
@@ -0,0 +1,133 @@
+import { NodeCG } from "nodecg/types/server";
+import { NodeCGIOCore } from "nodecg-io-core/extension";
+import { Service, ServiceProvider } from "nodecg-io-core/extension/types";
+import { Result, emptySuccess, error, success } from "nodecg-io-core/extension/utils/result";
+import { v4 as ipv4 } from "is-ip";
+import { v3 } from "node-hue-api";
+import Api = require("node-hue-api/lib/api/Api");
```

Comment:
This import is actually used but just as a typing. We can ignore the linter warning as it is actually used. 
```suggestion
// eslint-disable-next-line @typescript-eslint/no-unused-vars-experimental
import Api = require("node-hue-api/lib/api/Api");
```